app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,19 @@
+from .dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+
+    def __init__(
+        self, nickname: str, favourite_dish: str, skill_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._skill_level = skill_level
+
+    def player_info(self) -> str:
+        return (
+            f"Dwarf blacksmith {self.nickname} "
+            f"with skill of the {self._skill_level} level"
+        )
+
+    def get_rating(self) -> int:
```

Comment:
The `get_rating` method should have a return type annotation of `int` to match its implementation. Please update the method signature to reflect this.